app/include/zmk/hid.h

```diff
@@ -89,6 +90,37 @@ static const uint8_t zmk_hid_report_desc[] = {
     /* INPUT (Data,Ary,Abs) */
     HID_INPUT(0x00),
     HID_END_COLLECTION,
+
+    HID_USAGE_PAGE(HID_USAGE_GD),
+    HID_USAGE(HID_USAGE_GD_MOUSE),
+    HID_COLLECTION(HID_COLLECTION_APPLICATION),
+    HID_REPORT_ID(0x04),
```

Comment:
Should `0x04` (and the other report ID constants) be made into `#define`s and then used in `hid.c` in the report structs?

Reply:
Agreed. Also took the opportunity to refactor our main item values to use bit flag constants as well for clarity, which has been on my wishlist for this code.